src/compressed_tensors/quantization/quant_args.py

```diff
@@ -212,6 +212,17 @@ def pytorch_dtype(self) -> torch.dtype:
         else:
             raise ValueError(f"Invalid quantization type {self.type}")
 
+    def remove_defaults(self):
+        defaults = self.__fields__
+        result = {}
+        for field, value in dict(self).items():
+            if field == "observer_kwargs" and len(value) == 0:
+                continue
+            default = defaults[field].default
+            if value != default and value is not None:
```

Comment:
This assumes that the `None` will never be a non-default value. While this is currently the case, it will not always be the case, for example if we for some reason in the future set actorder to default to True, we want to serialize if the user sets actorder to `None`

Reply:
Or if we want to create an argument which defaults to a `non-None`, but has a `None` option